sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/ApplicationDetailsForCOE.vue

```diff
@@ -132,20 +94,27 @@ export default {
     },
   },
   setup(props: any) {
+    const { mapCOEChipStatus } = useCOE();
     const router = useRouter();
     const snackBar = useSnackBar();
     const initialData = ref({} as ApplicationDetailsForCOEAPIOutDTO);
     const items = ref([] as MenuType[]);
-    const denyCOEModal = ref({} as ModalDialog<FormIOForm & boolean>);
-    const confirmCOEModal = ref({} as ModalDialog<FormIOForm & boolean>);
-    const formioUtils = useFormioUtils();
+    const denyCOEModal = ref(
+      {} as ModalDialog<DenyConfirmationOfEnrollmentAPIInDTO | boolean>,
+    );
+    const confirmCOEModal = ref(
+      {} as ModalDialog<ApproveProgramInfoRequestModel | boolean>,
```

Comment:
Same comment from before (COE approval vs PIR approval), If I am missing something please ignore it.